server/utils/dateUtils.ts

```diff
@@ -30,7 +30,7 @@ export class DateFormats {
     if (options.format === 'long') {
       return format(date, 'cccc d MMMM y')
     } else {
-      return format(date, 'dd/LL/y')
+      return format(date, 'd MMM yy')
```

Comment:
This is a good shout - I think it's more in line with GDS standards, but should it be in its own commit?

Reply:
Done